src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -99,6 +104,19 @@ public Person toModelType() throws IllegalValueException {
         }
         final Email modelEmail = new Email(email);
 
+        boolean dobIsEmpty = (dob == ""); //check if DateOfBirth is empty
+        if (!DateOfBirth.isValidDate(dob)) {
+            throw new IllegalValueException(DateOfBirth.MESSAGE_CONSTRAINTS);
+        }
+
+        final DateOfBirth modelDob;
+
+        if (dobIsEmpty) {
+            modelDob = new DateOfBirth(); //create empty DateOfBirth object
```

Comment:
You can use the method `public static DateOfBirth empty() {...}` here to create empty date of birth object.